packages/shared/src/hooks/useTutorial.ts

```diff
@@ -3,7 +3,8 @@ import { useQuery, useQueryClient } from 'react-query';
 import usePersistentContext from './usePersistentContext';
 
 export enum TutorialKey {
-  SEEN_NEW_SQUAD_TOOLTIP_KEY = 'seenNewSquadTooltip',
+  SEEN_NEW_SQUAD_TOOLTIP = 'seenNewSquadTooltip',
+  SQUAD_ENABLE_NOTIFICATIONS = 'squadEnableNotifications',
   SHARE_SQUAD_POST = 'shareSquadPost',
   COPY_SQUAD_LINK = 'copySquadLink',
```

Comment:
I believe for enums' properties, we use PascalCase.

```suggestion
  SeenNewSquadTooltip = 'seenNewSquadTooltip',
  SquadEnableNotifications = 'squadEnableNotifications',
  ShareSquadPost = 'shareSquadPost',
  CopySquadLink = 'copySquadLink',
```

Reply:
We mix it now, but agree let's stick with PascalCase as it's most common actually.